content/en/mobile/_index.md

```diff
@@ -99,6 +99,97 @@ You can also specify a dashboard that opens by default when you tap on an SLOs w
 
 **Note**: If you do not specify a dashboard that opens by default, tapping an SLOs widget opens the Datadog app.
 
+#### Create an SLOs widget
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+- Long press on your home screen.
+- Tap the “+” button on the top left corner of the screen.
+- Search for "Datadog" widgets.
+- Select your prefered size (small shows one SLO and medium shows one SLO along with a visualized timeframe of its health).
+- Drag the widget to your desired, on-screen location.
+
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+- Long press on your home screen.
+- Tap the "Widgets" button on your home screen editor. If you have app shortcuts, it might appear as just an icon on the top right corner of the bubble.
+- Drag the widget to your desired, on-screen home screen.
+- Resize the widget to fit your preference; it will always shows one SLO. If you size the widget to take up the width of your mobile home screen, it will display the selected SLO along with a visualized timeframe of its health.
+
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Edit an SLOs widget
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+- Long press on the widget to configure.
+- Tap on "Edit Widget".
+- Tap on the "Choose" button next to the "SLO" label to choose an SLO to track.
+- Depending on the SLO chosen, a "Timeframe" label may appear; tap on the "Choose" button next to the "Timeframe" label to choose the SLO timeframe. 
+- Tap on the "Choose" button next to the "Dashboard to open" label to choose a dashboard that opens when the SLOs widget is tapped. 
```

Comment:
```suggestion
- Tap "Choose" next to the Dashboard to open label to choose a dashboard that opens when the SLOs widget is tapped. 
```

Reply:
```suggestion
- Tap "Choose" next to the "Dashboard to open" label to choose a dashboard that opens when the SLOs widget is tapped. 
```

I want to make clear to the reader that "Dashboard to open" is the name of the label. I am open to suggestions if there are other ways to make this clear without quotation marks.